DuckDuckGo/Autoconsent/AutoconsentUserScript.swift

```diff
@@ -291,13 +290,44 @@ extension AutoconsentUserScript {
         }
     }
 
+    @MainActor
+    func handlePopupFound(message: WKScriptMessage, replyHandler: @escaping (Any?, String?) -> Void) {
+        guard let messageData: PopupFoundMessage = decodeMessageBody(from: message.body) else {
+            replyHandler(nil, "cannot decode message")
+            return
+        }
+        Logger.autoconsent.info("Cookie popup found: \(String(describing: messageData))")
+        guard let url = URL(string: messageData.url),
+              let host = url.host else {
+            replyHandler(nil, "cannot decode message")
+            return
+        }
```

Comment:
these `guard`-s can be joined into one; 
I‘d add an `assertionFailure` to the `else` section to discover bugs early if something breaks

Reply:
wouldn't `assertionFailure` crash the app?
This message comes from a content script, and may be potentially affected by the page. Ignoring malformed data seems safe here, it should not leave the app in a weird state or anything